docs/manuals/core/versioning/production_mode.md

```diff
@@ -18,76 +18,217 @@ To convert an experiment version to a production, you can run the Taipy applicat
+Number of scenarios: 3
 
 $ taipy manage-versions --list
 Version number                         Mode                   Creation date
-1.0                                    Production (latest)    2023-01-25 13:00:05
+0.1                                    Production (latest)    2023-01-25 13:00:05
 325d0618-6f9e-459b-9597-48fa93a57a23   Experiment             2023-01-25 12:20:56
 26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development            2023-01-25 12:20:33
 ```
 
-In the example above, Taipy converted the version 1.0 to a production version before running it.
+In the example above, Taipy converted the version 0.1 to a production version before running it.
 
 Without explicitly providing the version name, the latest version of your application is used.
 Here is another example:
 
 ```console
-$ python main.py --experiment 2.0
-[2023-01-25 13:05:17,712][Taipy][INFO] job JOB_my_print_algo_ac79138a-4c3a-4560-bbd4-f4975083bf83 is completed.
-nb scenarios: 1
+$ python main.py --experiment 1.0
+[2023-01-25 13:05:17,712][Taipy][INFO] job JOB_example_algorithm_ac79138a-4c3a-4560-bbd4-f4975083bf83 is completed.
+Number of scenarios: 1
 
 $ taipy manage-versions --list
 Version number                         Mode                   Creation date
-2.0                                    Experiment (latest)    2023-01-25 13:05:17
-1.0                                    Production             2023-01-25 13:00:05
+1.0                                    Experiment (latest)    2023-01-25 13:05:17
+0.1                                    Production             2023-01-25 13:00:05
 325d0618-6f9e-459b-9597-48fa93a57a23   Experiment             2023-01-25 12:20:56
 26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development            2023-01-25 12:20:33
 
 $ python main.py --production
-[2023-01-25 13:06:00,871][Taipy][INFO] job JOB_my_print_algo_1fcb6feb-cef1-46e0-a818-4ae2e58df57d is completed.
-nb scenarios: 5
+[2023-01-25 13:06:00,871][Taipy][INFO] job JOB_example_algorithm_1fcb6feb-cef1-46e0-a818-4ae2e58df57d is completed.
+Number of scenarios: 5
 
 $ taipy manage-versions --list
 Version number                         Mode                   Creation date
-2.0                                    Production (latest)    2023-01-25 13:06:00
-1.0                                    Production             2023-01-25 13:00:05
+1.0                                    Production (latest)    2023-01-25 13:06:00
+0.1                                    Production             2023-01-25 13:00:05
 325d0618-6f9e-459b-9597-48fa93a57a23   Experiment             2023-01-25 12:20:56
 26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development            2023-01-25 12:20:33
 ```
 
-As you can see, we first created an experiment version named 2.0 and there is one scenario is
-attached to it. When listing the existing versions, we can see that 2.0 is the latest version used.
+As you can see, we first created an experiment version named 1.0 and there is one scenario is
```

Comment:
"and there is one scenario is attached to it" -> "and there is one scenario attached to it"
(although, I cannot "see" that)

Reply:
It's shown by the `Number of scenarios: 1` line.